src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/StorageInitializer.cs

```diff
@@ -1,7 +1,9 @@
 ﻿namespace NServiceBus.Unicast.Subscriptions.MessageDrivenSubscriptions
 {
+    using System.Collections.Generic;
     using NServiceBus.Features;
 
+    // This probably belongs to MsmqSubscriptionStorage and shouldn't be its own feature.
```

Comment:
Actually an anti-feature. Should be part of the MsmqSubscriptionStorage feature.